src/app/project-notifications/add-edit-project-notification/add-edit-project-notification.component.html

```diff
@@ -112,40 +124,77 @@ <h2 *ngIf="isEditing">Edit Project Notification</h2>
           </div>
         </div>
       </div>
-        <div *ngIf="myForm.value.decision && myForm.value.decision === 'Referred to Minister for Designation Decision'"
-          class="label-pair">
-          <label for="project" class="eAPProjectLabel">Associated Environmental Assessment Project*</label>
-          <div class="input-group">
-            <select name="project" id="project" formControlName="project" class="form-control">
-              <option value="">None</option>
-              <option *ngFor="let project of projects" [ngValue]="project._id">{{project.name}}</option>
-            </select>
-          </div>
+
+      <div *ngIf="myForm.value.decision && myForm.value.decision === 'Referred to Minister for Designation Decision'" class="label-pair">
+        <label for="project" class="eAPProjectLabel">Associated Environmental Assessment Project</label>
+        <div class="input-group">
+          <select name="project" id="project" formControlName="project" class="form-control">
+            <option value="">None</option>
+            <option *ngFor="let project of projects" [ngValue]="project._id">{{project.name}}</option>
+          </select>
         </div>
       </div>
+    </div>
+
+    <div class="flex-container">
+      <div class="label-pair lrg">
+        <label for="description">Description*</label>
+        <textarea name="description" id="description" formControlName="description" type="text"
+          class="form-control"></textarea>
+      </div>
+    </div>
+
+    <hr>
+    <p class="sub-headder">INFORMATION IN THIS SECTION <u>DOES NOT</u> DISPLAY PUBLICLY, FOR PROJECT TRACKING PURPOSES ONLY</p>
 
-      <div class="flex-container">
-        <div class="label-pair">
-          <label for="location">Location* </label>
-          <input name="location" id="location" formControlName="location" type="text" class="form-control" />
+    <div class="flex-container">
+
+      <div class="col-xl-7 col-lg-12 non-public-grouping-box">
+        <div class="notification-threshold-box col-12">
+          <strong>Notification Threshold* </strong>
+          <span class="info-text" >(Input a value, and the units the value is measured in, based on the triggered threshold)</span>
         </div>
-        <div class="label-pair">
-          <label for="latitude">Latitude*</label>
-          <input name="latitude" id="latitude" formControlName="latitude" type="text" class="form-control" />
+        <div class="label-pair half">
+          <input name="notificationThresholdValue" id="notificationThresholdValue" formControlName="notificationThresholdValue" type="text" class="form-control" />
```

Comment:
This value should only allow numeric values, currently accepts any string

Reply:
thanks, updated